set.mm

```diff
@@ -12580,10 +12587,18 @@ When the universe is infinite (as with set theory), such a
        These will be redeclared globally in the predicate calculus section. $)
     vx.tru $f setvar x $.
     vy.tru $f setvar y $.
-    $( Soundness justification theorem for ~ df-tru .  (Contributed by Mario
-       Carneiro, 17-Nov-2013.)  (Revised by NM, 11-Jul-2019.) $)
+    $( Soundness justification theorem for ~ df-tru .  Instance of
+       ~ monothetic .  (Contributed by Mario Carneiro, 17-Nov-2013.)  (Revised
+       by NM, 11-Jul-2019.)  (Proof shortened by BJ, 7-Sep-2022.) $)
```

Comment:
After adding a new theorem to Main, I used to update theorems profiting from that addition silently.  Was this just my habit?

Reply:
You're right, I'll remove that "shortened" tag.